roxie/ccd/ccdfile.cpp

```diff
@@ -895,7 +897,7 @@ class CRoxieFileCache : implements IRoxieFileCache, implements ICopyFileProgress
             ~AutoDisconnector() { if (f) disconnectRemoteFile(f); }
         private:
             IFile *f;
-        } autoDisconnector(f, autoDisconnect);
+        } autoDisconnector(f, autoDisconnect && useRemoteResources);
```

Comment:
Took a while to understand this change.  I assume it is because this is called from checkOpen and you are just about to copy the file, so you don't want it closed.  It would benefit from a comment.

Reply:
Added a comment